tests/testthat/test-use-rcpp.R

```diff
@@ -18,6 +18,35 @@ test_that("use_rcpp() creates files/dirs, edits DESCRIPTION and .gitignore", {
   expect_true(all(c("*.o", "*.so", "*.dll") %in% ignores))
 })
 
+test_that("use_rcpp_armadillo() creates Makevars files and edits DESCRIPTION", {
+  pkg <- scoped_temporary_package()
+  use_roxygen_md()
```

Comment:
Is this necessary?

Reply:
Per the decision to deprecate supporting manually edited `NAMESPACE` files in `usethis`, the use of `roxygen2` is required for `roxygen_ns_append()` function and only `use_roxygen_md()` setups the needed components. 

It may be beneficial to add a function solely enabling `roxygen2` via `use_roxygen()`. This would be better left to a separate PR.

Note: This is also the initial setup used by the `use_rcpp()` test.

Reply:
Got it, thanks